### PR TITLE
Ansible bump deps

### PR DESCRIPTION
**Pull request by fabianvf**
**Description of the change:**
Bring python and Ansible dependencies up to date

**Motivation for the change:**
Lots of work going on in the kubernetes and operator_sdk collection, want to make sure we keep up with those dependencies.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: changelog/fragments/ansible-dep-bump.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Update dependencies for Ansible-based Operators. Pulls in 0.12 of the
```
Comment (camilamacedo86) [inline changelog/fragments/ansible-dep-bump.yaml:5]:
```suggestion
      For Ansible-based Operators, update of dependencies. Pulls in 0.12 of the
```

Comment (camilamacedo86) [reply changelog/fragments/ansible-dep-bump.yaml:5]: Just nits. 
- According to the template, we ought to For Language-based ... Please, give a look at the template. 
- Also, we might need to create another one for the scaffolds using (ansible/v1) update of the dependencies and with the steps in the migration section to let the users are able to do that manually
- Then, it is missing `(`)` for the versions and the info from X to Y.

**Review by camilamacedo86 — APPROVED** [review]
Just nits regarding the fragment. Otherwise, 

lgtm

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**